### ISSUE TITLE
Create dedicated landing page for D7 EOL email campagin

### ISSUE THREAD

**Issue by jenlampton** [backdrop-ops/backdropcms.org#994]
We would like to create a dedicated landing page to use as the CTA for our Drupal 7 end-of-life email campaign.

This page should include: 
* some content from, and links to the [backdropcms.org/drupal-alternative](https://backdropcms.org/drupal-alternative) page
* links to the D7 EOL announcement
* links the Stanford project (or maybe a view with a teaser, as opposed to just a link)
* links to the proposed webinars 
* links to the migration pipeline project (or preferably the sign-up webform itself, if it is ready)

Below is some text we had removed from our email, which may be useful for the landing page.

If concern over budget is what's been stopping you from migrating to Drupal 9 or 10, then come check out our community. Ask our experts if Backdrop could work for your specific case. Ask your own developers to review the upgrade process, and compare the cost of a Backdrop upgrade to that of a Drupal migration.

Read about how the [Stanford University saved hundreds of thousands of dollars](https://backdropcms.org/news/why-we-moved-two-stanford-websites-to-backdrop-cms) by moving two of their more complex sites to Backdrop CMS instead of choosing the next version of Drupal.

**Comment by docwilmot** [comment]
Is anyone working on this yet? @bugfolder @cellear?

**Comment by cellear** [comment]
Sort of, but I didn't actually realize this issue was here.  This issue will help me focus...

**Comment by bugfolder** [comment]
If there's anything I can do to help, let me know, via issue or Zulip.

**Comment by docwilmot** [comment]
Thanks @cellear @bugfolder I'm eager to get this going. Opportunities await, IMO.

**Comment by docwilmot** [comment]
I would prefer if we could start the page by immediately selling Backdrop's easy upgrade path and mentioning D7s pending demise, rather than the current focus on the last 10yrs of Drupal. To me its a marketing page, and we need to immediately capture the site owners who click through to here, by giving them cause to be concerned (D7 may go away) and how we can help, in the first few lines.

In fact maybe even the page title should be less Drupal; Maybe "Convert your Drupal site easily to Backdrop"

**Comment by bugfolder** [comment]
Yeah, I agree with @docwilmot; you're kind of burying the lede. I'd suggest something like this:

> "Drupal 7 is rapidly approaching end-of-life. Backdrop provides a straightforward upgrade from your Drupal 7 site to the fast, stable, open-source and long-term-maintained Backdrop CMS. Over 2000 websites have already taken advantage of it!"

Key notions: 

* Get to the main message (2nd sentence) in as few words of context-setting as possible (1st sentence).
* Emphasize the key advantages in as few words as possible that get them across.
* Emphasize that others are already doing this (so you're not alone).

The rest of the page can flesh out the details, as you've done, but people should get the important part of the picture by the end of the first paragraph.

**Comment by jenlampton** [comment]
I love it. Can we make that whole first paragraph bigger and/or bolder
font, too?

**Comment by docwilmot** [comment]
Very much better, but still maybe a bit wordy, and I dont think the "History" blurb at the end adds much value. Plus we're giving suggestions to our potential clients to help them move away (use Wix or Wordpress!!??). My suggested edits:

- Backdrop is mature and stable.  [Backdrop](https://backdropcms.org/) at its core is still Drupal 7. But it’s an evolution of the Drupal 7 software, with hundreds of improvements for both website architects and content editors, and includes completely new functionality, like a powerful layout system and state-of-the-art configuration management. ~~All of that while still based on the same architecture that underpinned versions of Drupal prior to D8, and it is typically considerably easier to move a Drupal 7 site to Backdrop than it is to recent versions of Drupal.~~ **(this is easily implied by the rest of the points below)**
- Backdrop has a [built-in upgrade path from Drupal 7](https://backdropcms.org/upgrade-from-drupal). It will run on the same database as current Drupal 7 projects, so any configuration customizations that have been made to the Drupal 7 sites will be carried over when they are upgraded to Backdrop. Additionally, ~~the functionality of~~ over 70 of the most commonly used Drupal 7 contributed modules ~~has~~ have been included in Backdrop core, ~~so you will not need to install and configure these separately~~. These modules include Views, CKEditor, Pathauto, Redirect, and Token, to name a few. **It is typically considerably easier to move a Drupal 7 site to Backdrop than it is to recent versions of Drupal.** **(moved from first bullet)**
- Backdrop is secure! While still working closely with the Drupal community, Backdrop has its own [dedicated security team](https://backdropcms.org/leadership/security-team). We have been keeping Backdrop sites, and by extension the people that own and use them, safe for more than 8 years now. ~~Additionally, we have been taking the burden of maintainership of contributed projects off of single individuals and making it a responsibility of the community to step in and help out when needed.~~ **(we dont need to discuss contrib here IMO)**
- You will not need to re-learn everything or change the way you work. At the same time, site owners and content editors will not only not need to be re-trained - they will instead benefit from all the user experience, workflow, and usability improvements we've made over the years.  Developers familiar with Drupal 7 will feel right at home in Backdrop, since the API changes have been kept to the absolute minimum. ~~You will find working with the Backdrop interface to be a breeze (??? and much less annoying ???).~~ **(switched the sentence re site owners and the sentence about devs)**
- Designed for people who build websites.  The Backdrop team is continually working to make the out-of-the-box experience efficient and easy to understand. Backdrop is a perfect fit for the site builders - especially those that prefer to build sites with low-code/no-code. Backdrop is intended to be fully usable to people who don't generally write code, but it is also still a very powerful tool for the folks who do like to dive deeper into the code.
- ~~Drupal «Features» reinvented.  Configuration management is one of the hottest new tools in both Backdrop and Drupal. It takes the former [Features module](https://www.drupal.org/project/features) to the next level: any Backdrop configuration can be exported from one website and used on other projects. Never create a blog or an image gallery from scratch again –  reuse existing or custom "recipes" and refine what you have already created.~~
- ~~Powerful layout system.  Backdrop CMS took the best of Panels and Context modules in order to build a [flexible layout system](https://backdropcms.org/guide/layouts) with drag and drop functionality. Everything you see on the page, including the header and navigation, is in a responsive layout.~~ **(the above two bullet points are already in the first bullet, and we could simply hyperlink those to another page with more info)**
- Runs fast, even on shared hosting. Backdrop will always aim to keep the server requirements low to keep the cost of hosting a Backdrop site reasonable. 
- Here to stay. Backdrop CMS has been around for more than 8 years now, a period over which we have released 25 minor versions. Always on-time. Every 4 months. On top of that, we have been creating additional security and bug fix releases as needed. **(is this the same as first bullet tho??)**
- A supportive and vibrant open-source community. The majority of us either own, or work for organizations that run their sites on Backdrop or support customers that own Backdrop sites. So we have a vested interest for Backdrop to thrive. We are all stakeholders, and our goal is to make working with Backdrop fast, easy, cost-effective and sustainable ~~[and...???]. We want our business to be prosperous, and whether you are a site owner, a developer, or a digital agency, we want to help you do the same with your business.~~


FInally I would replace the whole "History" blurb with the statement" Officially support for Drupal 7 is [scheduled to end November 1, 2023](https://www.drupal.org/about/core/policies/core-release-cycles/drupal-7-roadmap-and-release-schedule).

**Comment by irinaz** [comment]
New page looks great https://backdropcms.org/upgrade-from-drupal-7, I think this issue can be closed
